contrib/exporters/core/pipeline.go

```diff
@@ -137,6 +145,11 @@ type Pipeline struct {
 
 // NewPipeline defines the pipeline elements
 func NewPipeline(cfg *viper.Viper) (*Pipeline, error) {
+	mangler, err := ManglerHandlers.Init(cfg, "mangle")
```

Comment:
I don't see anyone calling this mangler (or storing it in the Pipeline).

Reply:
called by secadvisor